src/main/java/com/jcabi/github/mock/MkEvent.java

```diff
@@ -112,11 +114,65 @@ public int compareTo(
         throw new UnsupportedOperationException("#compareTo()");
     }
 
+    /**
+     * Describes the event in a JSON object.
+     * @return JSON object
+     * @throws IOException If there is any I/O problem
+     * @todo #1063 When the event has a label, retrieve and include the
+     *  label's color too. MkIssueEvents.create() will also need to be
+     *  updated accordingly.
+     */
     @Override
     @NotNull(message = "JSON is never NULL")
-    public JsonObject json() {
-        return Json.createObjectBuilder().add("event", this.type).add(
-            "actor", Json.createObjectBuilder().add("login", "test").build()
-        ).add("created_at", new Github.Time().toString()).build();
+    public JsonObject json() throws IOException {
+        final JsonObject obj = new JsonNode(
+            this.storage.xml().nodes(this.xpath()).get(0)
+        ).json();
+        JsonObjectBuilder builder = Json.createObjectBuilder()
+            .add("id", this.num)
+            .add(
+                "url",
+                String.format(
+                    // @checkstyle LineLength (1 line)
+                    "https://api.jcabi-github.invalid/repos/%s/issues/events/%s",
+                    this.coords,
+                    this.num
+                )
+            )
+            .add("commit_id", JsonValue.NULL)
+            // @checkstyle MultipleStringLiteralsCheck (1 line)
+            .add("event", obj.getString("event"))
+            .add(
+                "actor",
+                Json.createObjectBuilder()
+                    // @checkstyle MultipleStringLiteralsCheck (1 line)
```

Comment:
@cvrebert let's make this string literal as a variable. maybe static and package visible so tests can use it?